rust/yarp-sys/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "yarp-sys"
+version = "0.1.0"
+edition = "2021"
+license-file = "../../LICENSE.md"
+repository = "https://github.com/ruby/yarp"
+description = "Rust bindings to Ruby's YARP parsing library"
+links = "yarp"
+authors = ["Steve Loveless <steve.loveless@gmail.com>", "Ian Ker-Seymer <hello@ianks.com>"]
+keywords = ["ruby", "parser", "ffi", "bindings"]
+categories = [
+    "api-bindings",
+    "development-tools::ffi",
+    "external-ffi-bindings",
+    "parsing",
+]
+rust-version = "1.59.0"
+
+[lib]
+crate_type = ["staticlib", "rlib"]
```

Comment:
I think this section can be removed. `staticlib` asks `rustc` to produce an `.a` archive. Why would we ask Rust to do that when I imagine the C toolchain in this repository can do the same thing?

Reply:
🤦 Good point!